pkg/constants/meta.go

```diff
@@ -10,6 +10,8 @@ const (
 	LabelAppCreatedBy = "app.kubernetes.io/created-by"
 	AppCreator        = "moco"
 
+	LabelHostName = "kubernetes.io/hostname"
```

Comment:
I think we can use `corev1.LabelHostname`.
https://github.com/kubernetes/api/blob/3a2d6b5bb7f84daa2125131865eb81b151448d5b/core/v1/well_known_labels.go#L20

Reply:
Thanks! I fixed: https://github.com/cybozu-go/moco/pull/471/commits/bef9388871be165040b78b4d277169df335aee70